File: api-posts/src/main/java/com/github/alexeysol/geekregimeapiposts/configurations/JpaConfiguration.java
package com.github.alexeysol.geekregimeapiposts.configurations;

import com.github.alexeysol.geekregimeapicommons.utils.search.SearchableRepositoryImpl;
import org.springframework.context.annotation.Configuration;
import org.springframework.data.jpa.repository.config.EnableJpaRepositories;

@Configuration
@EnableJpaRepositories(
    basePackages = "com.github.alexeysol.geekregimeapiposts.repositories",
    repositoryBaseClass = SearchableRepositoryImpl.class
)
public class JpaConfiguration {}

File: src/main/kotlin/com/github/alexeysol/geekregimeapiusers/configurations/JpaConfiguration.kt
package com.github.alexeysol.geekregimeapiusers.configurations

import com.github.alexeysol.geekregimeapicommons.utils.search.SearchableRepositoryImpl
import org.springframework.context.annotation.Configuration
import org.springframework.data.jpa.repository.config.EnableJpaRepositories

@Configuration
@EnableJpaRepositories(
    basePackages = ["com.github.alexeysol.geekregimeapiusers.repositories"],
    repositoryBaseClass = SearchableRepositoryImpl::class
)
class JpaConfiguration
